apps/calendar/locales/calendar.en-US.properties

```diff
@@ -269,8 +269,8 @@ busy[few] = {{n}} events
 busy[many] = {{n}} events
 busy[other] = {{n}} events
 present.ariaLabel = Today
-past-other-month = Other month
-future-other-month = Other month
+past-other-month.ariaLabel = Other month
```

Comment:
Here and below: need to rename the key to something new (l10n requirement for updating strings).